doc/install/install-ceph-gateway.rst

```diff
@@ -510,24 +510,24 @@ Execute the following steps:
 
 #. Add the following contents to the file::
 
-    import boto
```

Comment:
Btw, do **I** need to sign when I commit something like this? I am not familiar with the linux kernel's process of signage, which you guys seem to be using.